packages/react-components/src/theming/v9ThemeShim.ts

```diff
@@ -19,6 +19,13 @@ import { blackAlpha, whiteAlpha, grey, grey10Alpha, grey12Alpha } from './themeD
  */
 const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
   return {
+    colorBrandBackground3Static: palette.themeLighter,
```

Comment:
was this from the types update?

Reply:
yes they added new properties to this type. I am not sure these map correctly, I am not sure what the process was to make that decision before

Reply:
This should be fine....

Reply:
Adding colors is alright, it's a reccomended way to do this, [when we have v8 and v9 themes in the same app.](https://react.fluentui.dev/iframe.html?viewMode=docs&id=concepts-migration-from-v8-components-theme-migration--page#creating-compatible-themes)(this link also attached above this const)
The only thing, the colours should be taken from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/shims/ThemeShim/v9ThemeShim.ts (see the link above for more details)